src/content/docs.md

```diff
@@ -26,3 +26,4 @@ To update these cards, edit src/_data/docs_cards.yml.
   </Card>
 {% endfor -%}
 </div>
+<!-- Small Hacktoberfest edit -->
```

Comment:
This is not a valid edit. If you would like to work on a valid issue, do let us know and we can try to find some issues that match your interests.